src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServer1.java

```diff
@@ -131,6 +131,9 @@ public void run(RunConfiguration config) throws AppEngineException {
           + Joiner.on(",").withKeyValueSeparator("=").join(appEngineEnvironment));
     }
 
+    String gaeRuntime = getGaeRuntimeJava(isJava8(config.getServices()));
```

Comment:
This section kinda feels weird to me. Is there any way we can integrate this into line [113](https://github.com/GoogleCloudPlatform/appengine-plugins-core/blob/1d6a525faa0f4f36a2e482b838d8e2394b62a2cb/src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServer1.java#L113) where we're distinguishing between java7 and java8 already? That way we keep the java7/java8 branching in one place? It'll probably get rid of the `getGaeRuntimeJava` function.